package.json

```diff
@@ -1,9 +1,12 @@
 {
   "dependencies": {
+    "@emotion/styled": "^11.3.0",
+    "@mui/material": "^5.1.0",
```

Comment:
Do we need to do npm install <package> for this, or will "npm install" already install all of the dependencies?

Reply:
Oh I forgot to update Readme. You need to npm install @mui/material and @emotion/styled for it to work.